protocols/dydx/index.json

```diff
@@ -0,0 +1,8 @@
+{
+  "cname": "dydx",
+  "description": "DYDX is a governance token that allows holders to govern the dYdX Layer 2 perpetuals protocol.",
+  "path": "example",
+  "type": ["snapshot", "Aave Governance V2"]
```

Comment:
Can you please share the snapshot and aave-gov links in the comments of this PR ?

Reply:
https://snapshot.org/#/dydxgov.eth

The DydxGovernor contract was inspired by Aave. Minor changes have been made by dYdX. 
Governor is deployed at 0x7E9B1672616FF6D6629Ef2879419aaE79A9018D2 - https://etherscan.io/address/0x7e9b1672616ff6d6629ef2879419aae79a9018d2